scripts/runner-update-env.js

```diff
@@ -73,7 +73,14 @@ async function fetchRunners() {
 
         services = services.concat(filteredServices);
 
-        await sleep(1000);
+        if (!response.headers.get('ratelimit-remaining') || !response.headers.get('ratelimit-reset')) {
+            throw new Error('Unexpected API format: no rate limit headers');
```

Comment:
would it make sense to just have a warning and a high sleep delay if there is no headers. Since this is not something under our control we might still want the script to execute, even if slow

Reply:
Maybe? I figure since it's a simple script we're running ourselves that if something changes with the API we'd dig into it and update the script anyway. I figure failing fast gives us the easiest way to emphasize something weird is happening.

Reply:
I had hit merge when I saw this, but if you want me to update to add the sleep on missing headers bit I'm more than happy to do so.

Reply:
don't worry. not a big deal